suites/pacific/rgw/sanity_rgw.yaml

```diff
@@ -180,6 +180,16 @@ tests:
         config-file-name: test_swift_basic_ops.yaml
         timeout: 300
 
+  - test:
+      name: swift versioning copy tests
+      desc: restore versioned object in swift
+      polarion-id: CEPH-10646
+      module: sanity_rgw.py
+      config:
+        script-name: test_swift_basic_ops.py
+        config-file-name: test_swift_version_copy_op.yaml
+        timeout: 500
+
```

Comment:
IMO, we should not add the suite to sanity if it is already added to `tier-1`. @rakeshgm your thoughts?

Reply:
I have removed code from sanity suite